src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
```

Comment:
It's a good practice to add a comment to briefly explain the purpose of the CSS file and its corresponding component.